services/github/schemas/v1/build-list.yml

```diff
@@ -35,7 +35,7 @@ properties:
           description: Github revision associated with the build.
         state:
           type: string
-          enum: ['pending', 'success', 'error', 'failure']
+          enum: ['pending', 'success', 'error', 'failure', 'cancelled']
```

Comment:
auto-cancelled builds get this status now `cancelled`.
This is only `github_builds` internal representation and state